index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/00.Website/assets/css/styles-home-page.css" />
+    <title>My portfolio</title>
+</head>
+<body>
+    
+    <div class="wrapper">
+        
+       <header>
+            <nav class="nav-wrapper">
+            
+                <a href="#" class="link-hamburger-icon">☰</a>
+                <!-- <ul>
+                    <li><a href="#">Hello</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul> -->
```

Comment:
- [ ] [OPTIONAL] Kindly remove unused lines of code to keep your codebase cleaner.